ansible/group_vars/all/defaults.yaml

```diff
@@ -59,7 +59,7 @@ aws_images: []
 
 extra_images: []
 
-containerd_supplementary_images: ["ghcr.io/mesosphere/toml-merge:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.5.0"]
+containerd_supplementary_images: ["ghcr.io/mesosphere/toml-merge:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.5.3"]
```

Comment:
Why do we need both `ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0` and `v0.5.3`?

Reply:
I honestly does not have context about this. @dkoshkin any idea?

Reply:
This was the PR that added `v0.5.0` https://github.com/mesosphere/konvoy-image-builder/pull/1019
Konvoy is still using `v0.2.0` and the old API https://github.com/mesosphere/konvoy2/blob/051fdf94628a09028d0c4befe8d27b04876d37f7/pkg/capi/generator/templates/dynamic-credential-provider-config.yaml.gotmpl#L12

I think we should keep `v0.2.0` like you did here and cut another ticket to do the API migration in Konvoy.

Reply:
Thank you for the context